integration-tests/src/test/resources/features/AddressAPIHappyPath.feature

```diff
@@ -11,6 +11,10 @@ Feature: Address API happy path test
     # TXMA event
     Then TXMA event is added to the SQS queue containing device information header
 
+    # Addresses
+    When /addresses is called
```

Comment:
this should be `/addresses contain addresses from the shared claims in the personIdentityTable`
and is expected to be an array